Sources/SwifterSwift/UIKit/UIViewExtensions.swift

```diff
@@ -270,11 +270,14 @@ public extension UIView {
 
     /// SwifterSwift: Add shadow to view.
     ///
+    /// - Note: This method only works with non-clear background color, or set a `shadowPath`.
```

Comment:
What does "or set a shadowPath" means, if I understood correctly this is "or if the view has a shadowPath set"? If is that what you mean can you update to make it clear? :)

Reply:
Updated, thanks